src/main/java/wooteco/subway/dao/LineDao.java

```diff
@@ -32,7 +32,7 @@ public LineDao(NamedParameterJdbcTemplate jdbcTemplate) {
     }
 
     public Long save(LineCreateRequest line) {
```

Comment:
```suggestion
    public Long save(Line line) {
```

컨트롤러에서 받은 dto를 repository 영역까지 내리지 마세요.
repository에서 도메인 객체만 처리하도록 만들어보세요